ansys/mapdl/core/mapdl.py

```diff
@@ -2080,8 +2067,9 @@ def run(self, command, write_to_log=True, **kwargs):
             self._log.info(msg)
 
             # This very likely won't be recorded anywhere.
-            # But just in case, adding info as a comment
-            command = f"/COM, PyAnsys: {msg}"  # Using '!' makes the output of '_run' empty
+
+            # But just in case, I'm adding info as /com
```

Comment:
Don't attribute yourself to the code.  People can always access the blame of a file if they need to know who.

```suggestion
            # But just in case, add info as /COM
```